sites/platform/src/environments/_index.md

```diff
@@ -271,3 +274,23 @@ title=In the Console
 {{< /codetabs >}}
 
 The environment is redeployed and becomes available for use again.
+
+## Trigger actions on `push`
+
+You can push changes to your environment and trigger the following actions at the same time:
```

Comment:
We may want to explain what are `push-option`, users may not be familiar with it and don't understand that it's standard in git.